up42/auth.py

```diff
@@ -115,26 +139,29 @@ def _find_credentials(self) -> None:
         Sources the project credentials from a provided config file, error handling
         if no credentials are provided in arguments or config file.
         """
-        if self.project_id is None or self.project_api_key is None:
+        if self.credentials_id is None or self.credentials_key is None:
             if self.cfg_file is None:
-                raise ValueError("Provide project_id and project_api_key via arguments or config file!")
+                raise ValueError("Provide credentials via arguments or config file!")
 
             # Source credentials from config file.
             try:
-                with open(self.cfg_file) as src:
+                if self.auth_type == AuthType.PROJECT.value:
+                    logger.info(
+                        "Project based authentication will be deprecated soon."
+                        "Please follow the [authentication guidelines](https://sdk.up42.com/authentication/)."
```

Comment:
Nice!
I guess @up42/documentation will update this page in parallel when we release this?

Reply:
Yes actually this is the new doc page, @dustgalactic pointed me to that link

Reply:
https://up42.atlassian.net/browse/DOC-789